### PR TITLE
UDX-233: Document the flaky test indicator in the spec explorer

### PR DESCRIPTION
**Pull request by debrisapron**

### CODE REVIEW

File: content/guides/core-concepts/cypress-app.md [hunk 1]
```diff
@@ -92,6 +92,32 @@ the performance of your long-running tests.
...
+<strong class="alert-header">What is a flaky test?</strong>
+
+A test is considered to be [flaky](/guides/dashboard/flaky-test-management) when
+it can pass and fail across multiple retry attempts without any code changes.
+
+</Alert>
+
+Specs with flaky tests are flagged with an indicator beside the spec name. This
+allows you to focus your efforts on unreliable and problematic areas of your
+test suite.
```
Comment (pstakoun) [inline content/guides/core-concepts/cypress-app.md:108]: "This means you can easily discern which areas of your application might result in an unreliable user experience." - is this better phrasing? per @mattvCypress

**Comment by pstakoun** [comment]
cc @mattvCypress @Jess-Cypress

File: content/guides/core-concepts/cypress-app.md (hunk 1 above)
Comment (mattvCypress) [reply content/guides/core-concepts/cypress-app.md:108]: @pstakoun I like your proposal.

Comment (pstakoun) [reply content/guides/core-concepts/cypress-app.md:108]: lol, I copied this from your comment on the doc verbatim @mattvCypress

File: content/guides/core-concepts/cypress-app.md [hunk 2]
```diff
@@ -92,6 +92,32 @@ the performance of your long-running tests.
...
+
+The flake indicator will display if any tests in the given spec have been flaky
+in your latest runs (limited to 4). Hovering over the indicator will show a
+tooltip with the following analytics, based off the last 50 runs:
+
+- **Rate** (flaky runs divided by total runs)
+- **Severity** (flaky rate at a glance)
+- **Last flaky** (how many runs ago)
```
Comment (Jess-Cypress) [inline content/guides/core-concepts/cypress-app.md:116]: In the preview, the font here looks rather large...

**Review by pstakoun — APPROVED** [review]

**Pull request merged**